src/test/java/utils/TestUtils.java

```diff
@@ -67,19 +68,33 @@ public class TestUtils {
   public static final String WALLET_RPC_LOCAL_WALLET_DIR = MONERO_BINS_DIR;
   public static final String WALLET_RPC_ACCESS_CONTROL_ORIGINS = "http://localhost:8080"; // cors access from web browser
   
+  // monero wallet lws configuration (change per your configuration)
+  public static final int WALLET_LWS_PORT_START = 8443;
+  public static final int WALLET_LWS_ADMIN_PORT_START = 8444;
+  public static final String WALLET_LWS_DOMAIN = "localhost";
+  public static final String WALLET_LWS_ADMIN_DOMAIN = "localhost";
+  public static final String WALLET_LWS_URI = WALLET_LWS_DOMAIN + ":" + WALLET_LWS_PORT_START;
+  public static final String WALLET_LWS_ADMIN_URI = WALLET_LWS_ADMIN_DOMAIN + ":" + WALLET_LWS_ADMIN_PORT_START;
+
   // test wallet config
   public static final String WALLET_NAME = "test_wallet_1";
   public static final String WALLET_PASSWORD = "supersecretpassword123";
   public static final String TEST_WALLETS_DIR = "./test_wallets";
   public static final String WALLET_FULL_PATH = TEST_WALLETS_DIR + "/" + WALLET_NAME;
-  
+  public static final String WALLET_LIGHT_PATH = TEST_WALLETS_DIR + "/test_wallet_light"; 
   // test wallet constants
   public static final BigInteger MAX_FEE = BigInteger.valueOf(7500000).multiply(BigInteger.valueOf(10000));
   public static final MoneroNetworkType NETWORK_TYPE = MoneroNetworkType.TESTNET;
   public static final String LANGUAGE = "English";
-  public static final String SEED = "silk mocked cucumber lettuce hope adrenalin aching lush roles fuel revamp baptism wrist long tender teardrop midst pastry pigment equip frying inbound pinched ravine frying";
-  public static final String ADDRESS = "A1y9sbVt8nqhZAVm3me1U18rUVXcjeNKuBd1oE2cTs8biA9cozPMeyYLhe77nPv12JA3ejJN3qprmREriit2fi6tJDi99RR";
-  public static final long FIRST_RECEIVE_HEIGHT = 171; // NOTE: this value must be the height of the wallet's first tx for tests
+  //public static final String SEED = "silk mocked cucumber lettuce hope adrenalin aching lush roles fuel revamp baptism wrist long tender teardrop midst pastry pigment equip frying inbound pinched ravine frying";
+  //public static final String ADDRESS = "A1y9sbVt8nqhZAVm3me1U18rUVXcjeNKuBd1oE2cTs8biA9cozPMeyYLhe77nPv12JA3ejJN3qprmREriit2fi6tJDi99RR";
+  //public static final String PRIVATE_VIEW_KEY = "198820da9166ee114203eb38c29e00b0e8fc7df508aa632d56ead849093d3808";
+  //public static final long FIRST_RECEIVE_HEIGHT = 171; // NOTE: this value must be the height of the wallet's first tx for tests
+  public static final String SEED = "gang sunken soya nautical gone woozy pram imitate iceberg wade wives unknown anybody distance cigar fungal outbreak quick goggles voucher simplest suffice jive jive iceberg";
```

Comment:
Let's keep the previous test wallet keys so the test wallet doesn't need to be changed in order to test the light wallet stack. You can keep this wallet's keys as a local change in your environment.